### DISCUSSION TITLE
Add comments in between comics in read lists

### DISCUSSION THREAD

**Discussion by ch13696** [Kareadita/Kavita#2627] [Ideas]
### Idea Description

There are read lists with brief instructions in between certain issues of comics. Such as "only read this part of Strange Tales", "this series is completely optional", or "read this event before continuing". These comments are mainly in Master Reading Orders of Marvel and DC comics.

Below is an example from comicbookreadingorder.com of what I'm suggesting. You'll notice in between X-Men Vol. 2 and Cable & Deadpool that it says to read the Planet Hulk event before continuing. 

![Screenshot 2024-01-19 083813](https://github.com/Kareadita/Kavita/assets/5604614/b4c6074c-9d5e-4631-9045-87fbe3452d69)

Maybe it could be possible to add a clickable + symbol to add a comment next to each issue or maybe before or after? Or possibly a comment bar below each issue. 

I get not wanting to flood the read list UI, but the reason why it's important is that in later issues these events could be referenced or you could miss important story elements. That's why we need to add comments so users can know what they shouldn't miss.

### Duration of Using Kavita

2 years

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by majora2007** [comment]
This is an interesting concept actually and can help drive reading lists to be more than just a simple ordered list. Currently, I think all of the softwares have a very limited reading list implementation. 

I'd like to open up this discussion to expand reading lists to be more than just that. So other ideas, I would love to hear (or modifications on this).

**Comment by DieselTech** [comment]
The CBL project has considered this when going through lists that people might want to put notes in or otherwise document things. It's already part of our new .cbl specification that we are working on. Once we are closer to completion with it we will have it posted for public feedback.

**Reply by ch13696** [reply]
This is exciting to see. I can't wait to see this released.
